src/BuildingRegistry/Building/BuildingCommandHandlerModule.cs

```diff
@@ -21,6 +22,7 @@ public sealed class BuildingCommandHandlerModule : CommandHandlerModule
             Func<ISnapshotStore> getSnapshotStore,
             EventMapping eventMapping,
             EventSerializer eventSerializer,
+            IAddCommonBuildingUnit addCommonBuildingUnit,
```

Comment:
I think we should already look which applications will break because they have not registered the IAddCommonBuildingUnit. We have bumped into this before.

Reply:
There shouldn't be any as it is already required in the `BuildingUnitCommandHandlerModule`